Tests/UnitTests/Networking/Backend/BaseBackendTest.swift

```diff
@@ -34,10 +34,13 @@ class BaseBackendTests: TestCase {
         self.httpClient = self.createClient()
         let attributionFetcher = AttributionFetcher(attributionFactory: MockAttributionTypeFactory(),
                                                     systemInfo: self.systemInfo)
-        self.backend = Backend(httpClient: self.httpClient,
-                               apiKey: Self.apiKey,
-                               attributionFetcher: attributionFetcher,
-                               dateProvider: MockDateProvider(stubbedNow: MockBackend.referenceDate))
+        let authHeaders = type(of: self.httpClient).authorizationHeader(withAPIKey: Self.apiKey)
```

Comment:
I'm confused about the usage of `type(of:)` here. Did you mean to use `HTTPClient.authorizationHeader`?
The latter is known at compile time, this is relying on the type at runtime. And since most of our classes are pretty much `final`, it should be the same.